src/pages/tutorials/library.tsx

```diff
@@ -21,6 +24,28 @@ interface TutorialsLibraryPageProps {
 	layoutProps: Omit<SidebarSidecarLayoutProps, 'sidecarSlot' | 'headings'>
 }
 
+function TutorialLibrarySidebar() {
+	return (
+		<>
+			<div className="g-hide-with-mobile-menu">
+				<ConnectedTutorialLibraryFilters />
+			</div>
+			<div className="g-show-with-mobile-menu">
+				<SidebarNavList>
+					{/* copied from: src/layouts/base-new/index.tsx */}
+					<SidebarNavMenuItem item={{ heading: 'Main Menu' }} />
+					{generateTopLevelSubNavItems().map(
+						(item: $TSFixMe, index: number) => (
+							// eslint-disable-next-line react/no-array-index-key
+							<SidebarNavMenuItem item={item} key={index} />
+						)
+					)}
+				</SidebarNavList>
+			</div>
+		</>
+	)
+}
```

Comment:
I think we need this to render a `Sidebar` so that the markup is consistent and so that we properly render the `h2` for the Sidebar title. I see that this is copied from `base-new`, so that should be updated as well but that's outside the scope of this PR.

I would expect something like this, which makes the sidebar content match other top-level sidebars (other than there not being a level down mobile control button):

```tsx
function TutorialLibrarySidebar() {
	return (
		<Sidebar showFilterInput={false} title="Main Menu">
			<div className="g-hide-with-mobile-menu">
				<ConnectedTutorialLibraryFilters />
			</div>
			<div className="g-show-with-mobile-menu">
				<SidebarNavList>
					{generateTopLevelSubNavItems().map(
						(item: $TSFixMe, index: number) => (
							// eslint-disable-next-line react/no-array-index-key
							<SidebarNavMenuItem item={item} key={index} />
						)
					)}
				</SidebarNavList>
			</div>
		</Sidebar>
	)
}
```

Reply:
Ooh word, good call, will update!

Reply:
Updated!

Reply:
This looks a bit funny though 🤔
<img width="315" alt="Screen Shot 2022-09-07 at 6 59 57 PM" src="https://user-images.githubusercontent.com/1289701/189005265-3a00f76e-2d1a-4612-b87f-64b0064f719f.png">

Reply:
Hmmm this sidebar is tricky. Go ahead and leave as is!

Reply:
Leaving it without for now, but added a TODO comment 💭